tests/x11/reboot_gnome.pm

```diff
@@ -24,7 +24,7 @@ sub run() {
     if (get_var("SHUTDOWN_NEEDS_AUTH")) {
         assert_screen 'reboot-auth';
         wait_still_screen;
-        type_password;
+        type_string $testapi::password, max_interval => 5;
```

Comment:
should be save to rely on https://github.com/os-autoinst/os-autoinst/pull/636 now